_docs-sources/guides/build-it-yourself/0-landing-zone/0-intro/0-what-youll-learn-in-this-guide.md

```diff
@@ -0,0 +1,37 @@
+---
+pagination_label: Build it Yourself - Configure Your Accounts With Landing Zone
+---
+
+# What you’ll learn in this guide
+
+This guide will walk you through the process of configuring a production-grade AWS account structure, including how to manage multiple environments, users, permissions, and audit logging. We’ll also discuss how to implement a Landing Zone solution that lets you quickly spin up new AWS accounts that all implement a security baseline that enforces your company’s policies.
+
+This guide consists of four main sections:
+
+
+<div className="dlist">
+
+#### [Core concepts](../1-core-concepts/0-aws-account.md)
+
+An overview of the core concepts you need to understand to set up an AWS account structure, including AWS
+Organizations, IAM Users, IAM Roles, IAM Groups, CloudTrail, and more.
+
+#### [Production-grade design](../2-production-grade-design/1-the-root-account.md)
```

Comment:
Should this link to not go the `intro` section?

Reply:
✅ Good catch.